src/student_success_tool/features/pdp/student.py

```diff
@@ -82,3 +87,12 @@ def diff_gpa_term_1_to_year_1(
     year_col: str = "gpa_group_year_1",
 ) -> pd.Series:
     return df[year_col].sub(df[term_col])
+
+
+def change_pell_to_T_F(
+    df: pd.DataFrame, 
+    *, 
+    pell_col: str = "pell_status_first_year",
+) -> pd.Series:
+    # Convert 'Y' to True and 'N' to False
+    return df[pell_col].map({'Y': True, 'N': False})  
```

Comment:
This is a fine way to do it (`Series.replace()` is the other). Since we know that inputs are required to be one of Y/N/null, this is safe. That said, let's lint this line and specify the dtype, to avoid surprises down the line:

```suggestion
    return df[pell_col].map({"Y": True, "N": False}).astype("boolean")
```